dcpy/test/resources/product_metadata/colp_single_feature_package/metadata.yml

```diff
@@ -5,348 +5,367 @@ attributes:
   display_name: City Owned and Leased Property (COLP)
   each_row_is_a: City Owned Property
   tags:
-  - a
-  - b
+    - a
+    - b
 
 assembly:
-- id: csv_package
-  filename: csv_package.zip
-  type: Zip
-  contents:
-  - id: primary_csv
-    filename: colp_{{ version }}.csv
-  - id: colp_readme
-    filename: colp_readme.pdf
-  - id: colp_metadata
-    filename: colp_metadata.pdf
+  - id: csv_package
+    filename: csv_package.zip
+    type: Zip
+    contents:
+      - id: primary_csv
+        filename: colp_{{ version }}.csv
+      - id: colp_readme
+        filename: colp_readme.pdf
+      - id: colp_metadata
+        filename: colp_metadata.pdf
 
 custom: {}
 
 destinations:
-- id: socrata_prod
-  type: socrata
-  files:
-  - id: colp_readme
-    custom:
-      destination_use: attachment
-  - id: primary_shapefile
-    custom:
-      destination_use: dataset_file
-  custom:
-    four_four: fn4k-qyk2
-- id: socrata_unparsed
-  type: socrata
-  files:
-  - id: colp_readme.pdf
+  - id: socrata_prod
+    type: socrata
+    files:
+      - id: colp_readme
+        custom:
+          destination_use: attachment
+      - id: primary_shapefile
+        dataset_overrides:
```

Comment:
This is the only functional change in the file. The rest is accidental auto-formatting 🤦‍♂️